iosHackthonApp/iosHackthonApp/Login/LoginView.swift

```diff
@@ -0,0 +1,85 @@
+//
+//  LoginView.swift
+//  iosHackthonApp
+//
+//  Created by Raynelle Francisca on 16/11/2020.
+//  Copyright © 2020 orgName. All rights reserved.
+//
+
+import SwiftUI
+import Strings
+import Theming
+
+struct LoginView: View {
+	@State var email: String = ""
+	@State var password: String = ""
+	@State var loginSucessful: Bool = true
+	let coloredNavAppearance = UINavigationBarAppearance()
+
+	init() {
+		coloredNavAppearance.configureWithOpaqueBackground()
+		coloredNavAppearance.backgroundColor = UIColor(ColorManager.appPrimary)
+		coloredNavAppearance.titleTextAttributes = [.foregroundColor: UIColor.white]
+		coloredNavAppearance.largeTitleTextAttributes = [.foregroundColor: UIColor.white]
+		UINavigationBar.appearance().standardAppearance = coloredNavAppearance
+		UINavigationBar.appearance().scrollEdgeAppearance = coloredNavAppearance
+	}
+
+	var body: some View {
+		NavigationView {
+			ScrollView {
+				VStack {
+					Image("") // Add proper logo
+						.resizable()
+						.scaledToFit()
+						.frame(width: 150, height: 150, alignment: .center)
+						.border(Color.gray, width: 2)
+						.cornerRadius(10)
+				}
+				.padding()
+
+				VStack(alignment: .leading, spacing: 10) {
+					Group {
+						Text(Strings.Login.email)
+						TextField(Strings.Login.emailPlaceholder, text: $email)
+							.modifier(GreyTextFieldStyle())
+						Text(Strings.Login.password)
+						SecureField(Strings.Login.passwordPlaceholder, text: $password)
+							.modifier(GreyTextFieldStyle())
+						Spacer()
+
+						GeometryReader { geometry in
+							let backgroundColor = email.isEmpty || password.isEmpty ? ColorManager.gray: ColorManager.appPrimary
```

Comment:
lets shift the .isEmpty checking to a computed variable and then we can use it for both the background colour and whether the button is disabled

Reply:
Nice catch..Done